### PR TITLE
f-checkout@v0.79.0

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Fixed
- Retry modal now closes when errors / issues are returned and the user tries to dismiss the modal.

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by simonsandersje — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Comment by SachaZvetelman** [comment]
You'll need to update the version 🔢

**Comment by kevinrodrigues** [comment]
> You'll need to update the version 🔢

Version is correct I believe (package.json) I've updated the title tho if that's what you mean?

File: packages/components/organisms/f-checkout/src/components/ErrorDialog.vue [hunk 1]
```diff
@@ -3,7 +3,8 @@
         :is-open="isOpen"
         data-test-id="checkout-issue-modal"
         has-overlay
-        @close="handleClose">
+        v-bind="$attrs"
+        v-on="$listeners">
```
Comment (charliearlie) [inline packages/components/organisms/f-checkout/src/components/ErrorDialog.vue:7]: Nice. I guess this is where me being new to Vue got me. 🙈

Comment (kevinrodrigues) [reply packages/components/organisms/f-checkout/src/components/ErrorDialog.vue:7]: So it was actually the `v-if` that was breaking it mostly. The mega modal wasn't picking up the `hasOpened` value as that `if` [here](https://github.com/justeat/fozzie-components/pull/845/files#diff-3f7bda367e6ae2f5c141559c1aec5722d6606ee4d895f7d44e9837c20f7efb3cL4) stopped it being populated so the close event would never execute properly :)

File: packages/components/organisms/f-checkout/test/specs/component/f-checkout.component.spec.js [hunk 2]
```diff
@@ -157,7 +157,7 @@ describe('f-checkout component tests', () => {
         expect(checkout.isCheckoutErrorMessageDisplayed()).toBe(true);
     });
 
-    it('should close the checkout error when "Retry" is clicked', () => {
+    xit('should close the checkout error when "Retry" is clicked', () => {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/test/specs/component/f-checkout.component.spec.js:160]: Skipped as there's a problem with the way the form submission works, causing the modal to possibly open twice. I've spoke to @Festivejelly about it and he's improving the way this works so should resolve this soon.

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**